tests/test_simple_circuits.py

```diff
@@ -35,20 +35,21 @@ class SimpleCircuitsTest(BaseTest):
     """
 
     num_subsystems = 4
+    shots = 16 * 1024
```

Comment:
The number of shots exceeds some limit set by IBM Q and this test fails. Changing this to 8*1024 works.